contrib/gitian-build.py

```diff
@@ -11,10 +11,10 @@ def setup():
     if args.kvm:
         programs += ['python-vm-builder', 'qemu-kvm', 'qemu-utils']
     elif args.docker:
-        programs += ['docker.io']
+        programs += ['docker.io', 'docker-ce']
     else:
         programs += ['lxc', 'debootstrap']
-    subprocess.check_call(['sudo', 'apt-get', 'install', '-qq'] + programs)
+    subprocess.check_call(['sudo', 'apt-get', 'install', '-qq', '--ignore-missing'] + programs)
```

Comment:
Looks like the `--ignore-missing` is not doing anything for me on bionic:

```
# apt install git docker-ce --ignore-missing
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package docker-ce is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'docker-ce' has no installation candidate
# echo $?
100

Reply:
Check if the OS is Bionic instead